ranking/protocol.go

```diff
@@ -0,0 +1,89 @@
+package ranking
+
+import (
+	"strings"
+
+	"github.com/PretendoNetwork/nex-go"
+	_ "github.com/PretendoNetwork/nex-protocols-go"
```

Comment:
This is imported on some common protocols so that we can access the `DataHolder` structures. There aren't any of them for the Ranking protocol, so this can be removed